### PR TITLE
Adding my pin

### PR DESCRIPTION
**Pull request by Jupiter-Mars**
@githubteacher trying to merge my pin 
close #2037

### CODE REVIEW

**Comment by dyhun** [comment]
Hi @Jupiter-Mars if you change "githubhandle" to "githubHandle" and "longtitude" to "longitude" and you also need put these caracters "---" to the first and last line it will be good

**Comment by lassemaidalpedersen** [comment]
As @dyhun stated, you need the three dashes on the first and last line, as well as fixing your typos and capitalization.

Better late than never.

**Comment by hectorsector** [comment]
🎉 👏 @lassemaidalpedersen @dyhun you're correct

@Jupiter-Mars please let us know if you need help adding the dashes!

File: _pins/Jupiter-Mars.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubhandle: Jupiter-Mars
```
Comment (sbs205c5) [inline _pins/Jupiter-Mars.json]: insert three dashes ("---") as the first line
change "githubhandle" to githubHandle"

File: _pins/Jupiter-Mars.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+latitude: 37.338208
+longtitude: -121.886329
```
Comment (sbs205c5) [inline _pins/Jupiter-Mars.json]: insert three dashes ("---") as the last line

**Comment by VanceCarey** [comment]
@Jupiter-Mars 
Good luck on future projects!

**Comment by crichID** [comment]
@Jupiter-Mars I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/2037/commits/b00cdc434541fe2a8a9c6850913f7241430dc344. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by 12/22, I will go ahead and do it just to keep the repository clean!

**Pull request merged**